guides/source/security.md

```diff
@@ -693,6 +693,18 @@ Note the previous mentioned countermeasures are only available in model instance
 try `sanitize_sql()` elsewhere. _Make it a habit to think about the security consequences
 when using an external string in SQL_.
 
+Also, if you want to use SQL LIKE like below, it may not be enough because SQL LIKE characters (i.e., `%` and `_`) are not sanitized.
+
+```ruby
+Model.where("name LIKE ?", params[:name])
+```
+
+In this case, we can use `sanitize_sql_like`.
+
+```ruby
+Model.where("name LIKE ?", Model.sanitize_sql_like(params[:name]))
+```
```

Comment:
We should probably make these examples more realistic.  If you don't want pattern matching, why would you use `LIKE` here instead of `=`?

Reply:
I am sorry this equals to one using `=`.
Is it enough to rewrite the code like below? Or does more realistic story required?

~~~markdown
```ruby
Model.where("name LIKE ?", "#{params[:name]}%")
```
```ruby
Model.where("name LIKE ?", "#{Model.sanitize_sql_like(params[:name])}%")
```
~~~

Reply:
Yes, queries like that would be more realistic.